ui_testing/testcases/base_test.py

```diff
@@ -6,6 +6,7 @@
 from ui_testing.pages.login_page import Login
 from ui_testing.pages.testplan_page import TstPlan
 from ui_testing.pages.testunit_page import TstUnit
+from ui_testing.pages.analysis_page import SingleAnalysisPage
```

Comment:
no need to define  SingleAnalysisPage here

Reply:
Done

Reply:
@dianaeldemerdash I can see you didn't remove it yet